common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -235,7 +219,8 @@ nav.sequence-nav {
           }
         }
 
-        &:hover, &:focus {
+        &:hover,
+        &:focus {
```

Comment:
If we're going to do this type of small NIT work, I'd rather us do it a clean up-based PR that does it across the application. Mind saving this change for that and reverting?